lib/kadai-core/src/main/java/io/kadai/task/internal/TaskQuerySqlProvider.java

```
@@ -55,7 +55,7 @@ public static String queryTaskSummaries() {
         + "<if test=\"addAttachmentClassificationNameToSelectClauseForOrdering\">, "
         + "ac.NAME as ACNAME </if>"
         + "<if test=\"addWorkbasketNameToSelectClauseForOrdering\">, w.NAME as WNAME </if>"
-        + "<if test=\"joinWithUserInfo\">, u.LONG_NAME</if>"
+        + "<if test=\"joinWithUserInfo and lockResults == 0\">, u.LONG_NAME</if>"
```

This is silently disallowing the combination.
We could instead throw an exception in `TaskQueryImpl` at query-generation (pre execution).